packages/main/src/types/BadgeDesign.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * Defines badge design types.
+ *
+ * @readonly
+ * @enum {string}
+ * @public
+ * @author SAP SE
+ * @alias sap.ui.webc.main.types.BadgeDesign
+ */
+enum BadgeDesign {
+	/**
```

Comment:
js docs for Set1, Set2 and Set3 are wrong, explain them based on the design
e.g. "Generic indication colors Set1" or  "Generic indication colors recommended for Info Label in BTP" and so on

Reply:
discuss with UA